app/src/main/res/values/strings.xml

```diff
@@ -7,7 +7,9 @@
     <string name="enable_notification_access_summary">Required for scrobbling from Apple Music</string>
     <string name="label_now_playing">Now Playing Track</string>
     <string name="label_not_playing">Nothing!</string>
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mataku/SunsetScrob/blob/95692b33a9b982064e176bd5409023a78fb17a18/app/src/main/res/values/strings.xml#L9"></span>The resource <code>R.string.label_not_playing</code> appears to be unused</td>
    </tr>
  </tbody>
</table>